packages/diracx-web-components/app/layout.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+// Force an error if any components use dynamic functions or uncached data
+// as these won't work in the static export.
+export const dynamic = "error";
+
+// This is needed for Jest tests
```

Comment:
It is really used by Jest. The tests won't run without it with the error:
```
> jest --ci

Error: > Couldn't find any `pages` or `app` directory. Please create one under the project root
```

And sadly the only available options for the file location are `/app` or `/src/app` as it uses NextJS.

Reply:
It sounds like there is still a `NextJS` config file somewhere and I think I found it:
https://github.com/DIRACGrid/diracx-web/blob/a8067cd485bb4813196f9a403048fa657f11bb1d/packages/diracx-web-components/jest.config.mjs

This should probably be tailored for a react library instead of a `NextJS` application.